HousingResidentInformationAPI/V1/Infrastructure/housingContext.cs

```diff
@@ -1,11 +1,11 @@
 using Microsoft.EntityFrameworkCore;
 
-namespace UHResidentInformationAPI.V1.Infrastructure
+namespace HousingResidentInformationAPI.V1.Infrastructure
 {
 
-    public class UHContext : DbContext
+    public class HousingContext : DbContext
     {
```

Comment:
I think we should keep this to UHContext. As Emma pointed out before, our DB is UH so UHContext will be more suitable